core/src/App.html

```diff
@@ -1,120 +1,70 @@
+      on:statusChanged="{splitViewStatusChanged}" on:wcCreated="{splitViewWCCreated}"
+      disableBackdrop="{disableBackdrop}"></SplitView>
     {/if}
   </Backdrop>
   {#if showLoadingIndicator}
-  <div
-    in:fade="{{delay: 250, duration: 250}}"
-    out:fade="{{duration: 250}}"
-    class="fd-page spinnerContainer"
-    aria-hidden="false"
-    aria-label="Loading"
-  >
-    <div
-      class="fd-busy-indicator--m"
-      aria-hidden="false"
-      aria-label="Loading"
-      data-testid="luigi-loading-spinner"
-    >
+  <div in:fade="{{delay: 250, duration: 250}}" out:fade="{{duration: 250}}" class="fd-page spinnerContainer"
+    aria-hidden="false" aria-label="Loading">
+    <div class="fd-busy-indicator--m" aria-hidden="false" aria-label="Loading" data-testid="luigi-loading-spinner">
       <div class="fd-busy-indicator--circle-0"></div>
       <div class="fd-busy-indicator--circle-1"></div>
       <div class="fd-busy-indicator--circle-2"></div>
     </div>
   </div>
   {/if}
-  <TopNav
-    pathData="{navigationPath}"
-    pathParams="{pathParams}"
-    on:handleClick="{handleNavClick}"
-    on:resizeTabNav="{onResizeTabNav}"
-    on:toggleSearch="{toggleSearch}"
-    on:closeSearchResult="{closeSearchResult}"
-    on:handleSearchNavigation="{handleSearchNavigation}"
-    bind:isSearchFieldVisible
-    bind:displaySearchResult
-    bind:displayCustomSearchResult
-    bind:searchResult
-    bind:inputElem
-    bind:luigiCustomSearchRenderer__slot
-  />
+  <TopNav pathData="{navigationPath}" pathParams="{pathParams}" on:handleClick="{handleNavClick}"
+    on:resizeTabNav="{onResizeTabNav}" on:toggleSearch="{toggleSearch}" on:closeSearchResult="{closeSearchResult}"
+    on:handleSearchNavigation="{handleSearchNavigation}" bind:isSearchFieldVisible bind:displaySearchResult
+    bind:displayCustomSearchResult bind:searchResult bind:inputElem bind:luigiCustomSearchRenderer__slot />
   {#if !(hideNav||hideSideNav)}
-  <LeftNav
-    pathData="{navigationPath}"
-    pathParams="{pathParams}"
-    on:handleClick="{handleNavClick}"
-    on:resizeTabNav="{onResizeTabNav}"
-  />
+  <LeftNav pathData="{navigationPath}" pathParams="{pathParams}" on:handleClick="{handleNavClick}"
+    on:resizeTabNav="{onResizeTabNav}" />
   {/if}
   {#if (tabNav && !hideNav)}
-  <TabNav
-    pathData="{navigationPath}"
-    pathParams="{pathParams}"
-    on:handleClick="{handleNavClick}"
-    resizeTabNavToggle="{resizeTabNavToggle}"
-  />
+  <TabNav pathData="{navigationPath}" pathParams="{pathParams}" on:handleClick="{handleNavClick}"
+    resizeTabNavToggle="{resizeTabNavToggle}" />
   {/if}
 </div>
 
 <script type="text/javascript">
   import Alerts from './Alerts.html';
   import ConfirmationModal from './ConfirmationModal.html';
   import Modal from './Modal.html';
-  import UserSettingsDialog from './UserSettingsDialog.html';
+  import UserSettingsDialog from './UserSettingsDialog.html'
```

Comment:
```suggestion
  import UserSettingsDialog from './UserSettingsDialog.html';
```
Should we use ; at the end?